src/main/java/org/sagebionetworks/web/client/widget/entity/EntityPageTopViewImpl.java

```diff
@@ -237,6 +241,17 @@ public EntityPageTopViewImpl(Binder uiBinder,
 		initProjectLayout();
 
 		initClickHandlers();
+
+		// use this callback to update wikiPageId
+		CallbackP<String> wikiReloadHandler = new CallbackP<String>(){
+			@Override
+			public void invoke(String wikiId) {
+				wikiPageId = wikiId;
```

Comment:
this could be a single call back to the presenter (caching wiki page id and bundle there)